scripts/makebumpver

```diff
@@ -348,6 +348,7 @@ class MakeBumpVer:
         git_range = "%s-%s-%s.." % (self.name, self.version, self.release)
         proc = run_program(['git', 'log', '--pretty=oneline', git_range])
         lines = filter(lambda x: x.find('l10n: ') != 41 and
+                       x.find('Merge pull request') != 41 and
                        x.find('Merge commit') != 41 and
                        x.find('Merge branch') != 41,
```

Comment:
Wouldn't it be better to just let git ignore all merge commits by using the ``--no-merges`` option?